### PR TITLE
Fix prow monitoring stack

### PR DESCRIPTION
**Pull request by chaodaiG**
Fix a couple of errors introduced in https://github.com/GoogleCloudPlatform/oss-test-infra/pull/678

### CODE REVIEW

File: prow/oss/cluster/monitoring/mixins/prometheus/prometheus.libsonnet [hunk 1]
```diff
@@ -10,6 +10,4 @@
 (import 'stale_alerts.libsonnet') +
 (import 'tide_alerts.libsonnet') +
 (import 'prober_alerts.libsonnet') +
-(import 'boskos_alerts.libsonnet') +
-(import 'plank_alerts.libsonnet') +
```
Comment (fejta) [inline prow/oss/cluster/monitoring/mixins/prometheus/prometheus.libsonnet:14]: We don't want plank alerts?

**Review by fejta — APPROVED** [review]
/hold

File: prow/oss/cluster/monitoring/mixins/prometheus/prometheus.libsonnet (hunk 1 above)
Comment (chaodaiG) [reply prow/oss/cluster/monitoring/mixins/prometheus/prometheus.libsonnet:14]: The plank alert in k8s prow is a heartbeat job, which doesn't exist in oss prow yet. Will add the plank alert once the heartbeat job is up.

k8s plank alert: https://github.com/kubernetes/test-infra/blob/master/config/prow/cluster/monitoring/mixins/prometheus/plank_alerts.libsonnet#L17

**Pull request merged**